pyproject.toml

```diff
@@ -61,7 +62,7 @@ griffe = "1.1.1"
 towncrier = "~23.6.0"
 to-json-schema = "*"
 jsonschema = "*"
-moto = "~2.2.11"
+moto = "~4.2.14"
```

Comment:
That's a big jump, but I guess our usage is pretty narrow - no breaking changes we need to be aware of?

Reply:
This is the first release that supports Python 3.12. I could have jumped to 5.x. No breaking changes that I found, and the tests pass. We are using this for mocking AWS, and pretty basic at that.